docs/articles/nunit/writing-tests/attributes/parallelizable.md

```diff
@@ -4,11 +4,20 @@ uid: parallelizableattribute
 
 # Parallelizable
 
+Added in **NUnit 3.7.**
+
 The **ParallelizableAttribute** is used to indicate that a test and/or its descendants may be run in parallel with other tests. By default, no parallel execution takes place.
 
 When used without an argument, **Parallelizable** causes the test fixture or method on which it is placed to be queued for execution in parallel with other parallelizable tests. It may be used at the assembly, class or method level.
 
-The constructor takes an optional **ParallelScope** enumeration argument (see below), which indicates whether the attribute applies to the item itself, to its descendants or both. It defaults to **ParallelScope.Self**. The Scope may also be specified using the named property **Scope=**.
+> [!WARNING]
+> When tests are run in parallel, you are responsible for the thread safety of your tests. Tests that run at the same time and modify instance fields or properties without locks will cause unexpected behavior as they would in any multi-threaded program. If you are using fields or properties between parallel tests consider using the **FixtureLifeCycleAttribute** to construct a new instance of a test fixture (class) for each test that is run.
+
+The constructor takes an optional **ParallelScope** enumeration argument (see below), which indicates whether the attribute applies to the item itself, to its descendants or both. It defaults to **ParallelScope.Self**. The Scope may also be specified using the named property **scope=**, for example;
```

Comment:
If it's a settable property, it would be `Scope =`. If it's a constructor parameter, it would be `scope:`. Both might be options at the same time.